DicomRtImportExport/Logic/vtkSlicerDicomRtReader.cxx

```diff
@@ -77,18 +77,53 @@ class vtkSlicerDicomRtReader::vtkInternal
   public:
     BeamLimitingDeviceEntry()
       :
-      Distance(-1.),
-      NumberOfLeafJawPairs(0)
+      SourceIsoDistance(400.),
+      NumberOfPairs(0)

```

Comment:
Why did you rename this variable to have a less informative name? I'm actually curious, so if it's justifiable then no need to change, but in general we prefer longer but unambiguoug names. Not every reader of the code is an expert in every aspect of RT.

Reply:
I will return previous names.

Reply:
Thanks! You also changed some names to be more informative, so those should be kept (like the SourceIsoDistance)

Reply:
SourceIsoDistance parameter has different physical meaning for RTPlan and RTIonPlan but the same DICOM tag (300A,00BA) 

1. For RTPlan its "Source to Beam Limiting Device Distance";
2. For RTIonPlan its "Isocenter to Beam Limiting Device Distance".

In any case the name of the parameters isn't obvious, but i will keep it.